ctagsdotd/vbnet.ctags

```diff
@@ -0,0 +1,12 @@
+# See additional-language.ctags for maintaining this file
+--langdef=vbnet
+--langmap=vbnet:.vb
+--regex-vbnet=/^(?i)\s*(?:(?:Public|Private|Protected|Friend|Overloads|Overrides|Overridable|NotOverridable|MustOverride|Shared|Shadows|Async|Iterator)\s+)*\s*(?:Function|Sub)\s+([A-Z_0-9\[\]]*)\(.*?\)\s*(?:As\s+[\w.]+)?/\1/m,method/
+--regex-vbnet=/^(?i)\s*(?:(?:Default|Public|Private|Protected|Friend|Overloads|Overrides|Overridable|NotOverridable|MustOverride|Shared|Shadows|WriteOnly|ReadOnly|Iterator)\s+)*\s*(?:Property)\s+([A-Z_0-9\[\]]*)\s+As\s+/\1/p,property/
+--regex-vbnet=/^(?i)\s*(?:(?:Public|Private|Protected|Friend|Shared|Shadows|Custom)\s+)*\s*(?:Event)\s+([A-Z_0-9\[\]]*)/\1/e,event/
```

Comment:
Looking at the output of `ctags --list-kinds-full` from `universal-ctags`, I see that it has `E event`, `e enumerator` and `g enum`. So I recommend following the same here, instead of using `e,event`, `e,enum` (Based on the distinct short codes in the `ctags --list-kinds-full` output, I suspect using the same short code for different kinds might cause some problem, but I haven't tested it.)